packages/components/src/InputValidation/InputValidation.css

```diff
@@ -11,6 +11,8 @@
 
 .message > svg {
   margin-right: var(--space-smaller);
+  flex-shrink: 0;
+  align-self: flex-start;
```

Comment:
Code looks good. Not sure if we want the `align-self` though. I will let @fagnanm chime in with his thoughts before approving

Reply:
I'm good with the change. Flex-start will make it align with the top line instead of the middle of the block which I believe looks better.

Reply:
Fair enough. I personally like the center aligned but I will defer to you on that and we can change it in the future